tests/functional/specs/pyi_osx_event_forwarding.spec

```diff
@@ -0,0 +1,48 @@
+# -*- mode: python -*-
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+import os
+
+app_name = 'pyi_osx_event_forwarding'
+custom_url_scheme = os.environ.get('PYI_CUSTOM_URL_SCHEME', 'pyi-test-app')
+custom_file_ext = os.environ.get('PYI_CUSTOM_FILE_EXT', 'pyi_test_ext')
+
+a = Analysis(['../scripts/pyi_pyqt5_log_events.py'])
```

Comment:
I have no idea how all this works -- tbh I just copied the other extant test that did it this way.  What is `SRCPATH`?  Bear in mind I am a nub with pyinstaller aside from having beaten it into submission to work with my app these past 2 years -- I am still green.  Can you tell me exactly what to change?  I'll change it no problem!

Reply:
Or, if you can just explain in a bit more detail what exactly the weirdness is here?  Just ... I really don't know what .spec files have defined or not or what the idiomatic thing would be -- I literally copy-pasted the other OSX appleevent test that was there... but I am happy to change it.. I ... just do not know what exactly to avoid or do..?

Reply:
Should I just replace:  `'../scripts/pyi_pyqt5_log_events.py'`  with `SRCPATH`? Will that do the trick?   I'll try it a bit later and see.. if I don't hear back, and push a commit if that works.

Reply:
`SPECPATH` gives the folder that the spec file is in (so `tests/functional/specs` in this case). See [this](https://pyinstaller.readthedocs.io/en/stable/spec-files.html#globals-available-to-the-spec-file) corner of the docs.
```suggestion
a = Analysis([os.path.join(os.path.dirname(SPECPATH), 'scripts/pyi_pyqt5_log_events.py')])
```

Reply:
Oh I misread it. Ok I will add this now and if it's green I will push this along with the change(s) requested by @rokm .

Reply:
Pushed a commit that has this suggested change.